modules/installation/02-basics.qmd

```diff
@@ -0,0 +1,108 @@
+
+- Source pane
+- Console pane
+- Environment pane
+- Outut pane
+
+The first time you open the RStudio, you will be greeted by the three panes console, environment, and output. We will talk about the Source pane later in the chapter.
+
+```{r}
+knitr::include_graphics(here::here("modules/installation/pictures/02-basics/RStudio_interface_3p.png"))
+```
+
+On the top right-hand side, you have the **Environment** pane, which includes the Environment, History, Connections, Build, and Version Control System (VCS) tabs. We will focus on the Environment tab, which shows the currently saved R objects.
+
+**Output** pane, located at the bottom-right of the app, is used to display various outputs such as plots, HTML content, or files directory. You can interact with it as well, not just for shows. For example, in the "Files" tab, you can add new folders, new blank files, delete/rename, and more.
+
+On the left-hand side, you have the **Console** pane, which is the place where you can type out the command written in the R language and execute it immediately on your computer.
+
+:::{.callout-important}
+You can run the R code inside the **Console**, but once you close RStudio, you will lose all the code.
+:::
+
+To make your work reproducible, you should write and store all the code in the R Script (.R). It will show up in the **Source** pane, allowing you to view and edit various code-related files.
+
+```{r}
+knitr::include_graphics(here::here("modules/installation/pictures/02-basics/RStudio_interface_4p.png"))
+```
+
+
+# R Script
+
+There are multiple ways of adding the R Script. To get started, click on the *paper with a plus* symbol and choose R Script. 
+
+```{r}
+knitr::include_graphics(here::here("modules/installation/pictures/02-basics/RStudio_newfile.png"))
+```
+
+In an R Script, you can write code and add comments. Also, you can run each line of code by pressing **ctrl + enter/return** on Windows and Linux or **cmd + enter/return** on Macs. The code messages will shown in the **Console**.
+
+:::{.callout-tip}
+There is a shortcut key for creating a new R Script, which is **shift + ctrl + N** for Windows and Linux or **shift + cmd + N** for Macs.
+:::
+
+
+# R packages installation and loading
+
+An R package bundles all useful functions, help files, and data sets into a package. The installation process of the packages is like purchasing a book (package) and storing it in a bookshelf (library folder), and you grab it when you want to read it.
+
+Let's start by purchasing a book (install the package). Through the interface, navigate to the "Packages" tab in the **Output** pane. It will show you what packages you currently have. Next, click "Install", and you should be able to see the pop-up window. Type the package name you want to install in the "Packages" field. You need to make sure that you install all the dependencies by clicking on the box labeled "Install dependencies". Then click "Install", you are now ready to start reading a book.
```

Comment:
> purchasing a book (install the package) [...] you are now ready to start reading a book.

I think the analogy isn't needed here, especially if the emphasis is on the book and not the package.

Reply:
Some more work on the analogy could make it work if you think it is helpful, especially with graphics (books & hex stickers, bookshelf & system library, etc.)